gecko_tests/index.rst

```diff
@@ -1,16 +0,0 @@
-.. _gecko_tests:
-
-===========
-Gecko Tests
-===========
-
-Gecko tests documentation
-
-.. toctree::
-    :maxdepth: 2
-    :glob:
-
-    mochitest_xpcshell_manifest_keywords
-    reftest_manifest_keywords
```

Comment:
I couldn't find this stuff in firefox source docs.  If it is still relevant, should it be moved there, or did I just miss it?